wrappers/services.go

```diff
@@ -139,11 +139,9 @@ func stopService(sysd systemd.Systemd, app *snap.AppInfo, inter interacter) erro
 
 	switch app.DaemonScope {
 	case snap.SystemDaemon:
-		stopErrors := []error{}
-		for _, service := range extraServices {
-			if err := sysd.Stop([]string{service}, tout); err != nil {
-				stopErrors = append(stopErrors, err)
-			}
+		var stopErrors error
+		if 0 < len(extraServices) {
```

Comment:
(nitpick) that style is a bit unusual - don't we normally write: `if len(extraSerices) > 0 {` ?